### DISCUSSION TITLE
improving tabular display on doctr benchmarks

### DISCUSSION THREAD

**Discussion by nikokks** [mindee/doctr#1628] [Ideas]
Hi =) 
we can see that in that link, the tabulars data are not aligned.
https://github.com/mindee/doctr/blob/9c92f5cbd0bf64706672c29bf2d43117815c1794/docs/source/using_doctr/using_models.rst#L34
We could use mermaid to improve the displayed tabs.
I can do it if you are interested =)

**Comment by felixdittrich92** [comment]
Hey :)
Sure always happy about doc improvements ^^
Do you have an example with a mermiad table to see how it looks ? :)

**Reply by nikokks** [reply]
doctr can use this example 

| En-tête 1 | En-tête 2 | En-tête 3 |
|:----------|:---------:|----------:|
| gauche    | centré    | droite    |
| gauche    | centré    | droite    |

**Reply by nikokks** [reply]
it is done like that
```
| En-tête 1 | En-tête 2 | En-tête 3 |
|:----------|:---------:|----------:|
| gauche    | centré    | droite    |
| gauche    | centré    | droite    |
```

**Reply by felixdittrich92** [reply]
Yeah looks nice :+1: 
Feel free to open a PR :D
